src/twisted/newsfragments/8375.feature

```diff
@@ -0,0 +1 @@
+twisted.application.internet.ClientService now accepts a function to initialize a connection as the onNewConnection argument
```

Comment:
Missing a full stop.
Is the function used to initialize the connection or is called when the connection is initialized?